src/Models/User.php

```diff
@@ -114,7 +114,7 @@ class User extends SentryUser implements HasPresenter
      */
     public function revisions()
     {
-        return $this->hasMany('GrahamCampbell\Credentials\Models\Revision');
+        return $this->hasMany(\Config::get('credentials.revision'));
```

Comment:
I'm not following you here.  In my case, I've set `credentials.revision` to my own FQCN, not an alias, and it works fine with my change.  I guess your class could be added as a default value, in case the user hasn't properly instaled the config file or something.